### PR TITLE
Docs/prettier guide

### PR DESCRIPTION
**Pull request by Fletch99**
# Description

This is a guide to get developers on board with the process a Thoth Tech of using Prettier and Vale for correcting spelling ang grammar mistakes as well as having consistent formatting. 

## Type of change

- [x] Documentation (new)

# How Has This Been Tested?

Prettier and Vale has been run on this to fix any spelling error and formatting 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review from @ShaeChristmas  on the Pull Request

### CODE REVIEW

File: docs/learning/useful-resources/setup-prettier-and-vale.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Setup Prettier and Vale for VS Code
+
+Everyone at Thoth Tech should be using both
+[Prettier](https://prettier.io/), as well as [Vale](https://marketplace.visualstudio.com/items?itemName=errata-ai.vale-server). These allow for Pull Requests to run smoothly and allow Thoth Tech to have a consistent format, free from grammar and spelling issues. 
+
+
+
+## Setting up Prettier
+To install Prettier do the following:
+1. In Visual Studio code go to `Extensions` page.
```
Comment (ShaeChristmas) [inline docs/learning/useful-resources/setup-prettier-and-vale.md]: Do we have methods for using Prettier without VSC? Maybe a requirements section or something similar would help

File: docs/learning/useful-resources/setup-prettier-and-vale.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Setup Prettier and Vale for VS Code
+
+Everyone at Thoth Tech should be using both
+[Prettier](https://prettier.io/), as well as [Vale](https://marketplace.visualstudio.com/items?itemName=errata-ai.vale-server). These allow for Pull Requests to run smoothly and allow Thoth Tech to have a consistent format, free from grammar and spelling issues.
```
Comment (ShaeChristmas) [inline docs/learning/useful-resources/setup-prettier-and-vale.md]: Vale mentioned, but no set up for vale in this document.

Comment (ShaeChristmas) [reply docs/learning/useful-resources/setup-prettier-and-vale.md]: Dont mind me, was looking at an older version of the doc

File: docs/learning/useful-resources/setup-prettier-and-vale.md (hunk 1 above)
Comment (Fletch99) [reply docs/learning/useful-resources/setup-prettier-and-vale.md]: @ShaeChristmas Added a requirement section outlining VS Code is needed.

**Review by ShaeChristmas — APPROVED** [review]
Looks good!

**Pull request merged**